src/include/timestamp_inline.h

```diff
@@ -120,7 +120,7 @@
     do {                                            \
         (ta)->newest_start_durable_ts = WT_TS_NONE; \
         (ta)->newest_stop_durable_ts = WT_TS_NONE;  \
-        (ta)->oldest_start_ts = WT_TS_MAX;          \
+        (ta)->oldest_start_ts = WT_TS_NONE;         \
```

Comment:
Was this a bug? As in did we used to incorrectly init that value to `WT_TS_MAX`?

Reply:
No. it is not a bug. To improve the calculation of avoiding 0 as a valid timestamp, I need to change the initial value.